plugins/modules/identity/ipa/ipa_config.py

```diff
@@ -263,6 +277,7 @@ def ensure(module, client):
         ipapwdexpadvnotify=module.params.get('ipapwdexpadvnotify'),
         ipasearchrecordslimit=module.params.get('ipasearchrecordslimit'),
         ipasearchtimelimit=module.params.get('ipasearchtimelimit'),
+        ipaselinuxusermaporder=module.params.get('ipaselinuxusermaporder'),
```

Comment:
If the above is true, you could pass in here the transformed form if it is not `None`.

Reply:
Instead of transforming it here, I updated the `get_config_dict` function to transform it. Which also transforms e.g. `ipagroupsearchfields` and `ipausersearchfields` already.